PayButton/paybutton-server
# [#253] feat/send-email

Related to #253





Description
---
1. Implements the email trigger.
2. Fixes bug where trigger couldn't be updated


Test plan
---
1. First put the new variables in `.env.local` and `paybutton-config.json`
2. Restart the containers
3. Set up a trigger to an email you own on a button with an address you own
4. Make a payment to that address
5. Ensure the email comes, the HTML formatted


Finally, check if the trigger email can be updated normally.
